guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -15,46 +15,65 @@ New hosts are provisioned with a boot disk or PXE-less discovery image that {Pro
 
-In {Project} provisioning, the PXE loader which is a DHCP file name that defines which file to load through TFTP during PXE provisioning. For BIOS system, the file is `pxelinux.0`. For UEFI systems, the file is `grub2/grubx64.efi`.
+In {Project} provisioning, the PXE loader option defines DHCP filename option to use during provisioning. For BIOS system, use option "PXELinux BIOS" which instructs a provisioned node to download `pxelinux.0` file over TFTP. For UEFI systems, use option "PXEGrub2 UEFI" which instructs a TFTP client to download `grub2/grubx64.efi` file.
 
 For BIOS provisioning, you must associate a PXELinux template with the operating system.
 
 For UEFI provisioning you must associate a PXEGrub2 template with the operating system.
 
 If you associate both PXELinux and PXEGrub2 templates, {ProjectX} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
 
-
 [[Provisioning_Bare_Metal_Hosts-Prerequisites_for_Bare_Metal_Provisioning]]
 === Prerequisites for Bare Metal Provisioning
 
 The requirements for bare metal provisioning include:
 
-  * Synchronized content repositories for the version of Red Hat Enterprise Linux that you want to use. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red Hat Repositories] in the _Content Management Guide_.
+ifeval::["{Build}" == "foreman"]
+  * The installation media that you require for the operating systems you want to use to provision using Amazon EC2.
+endif::[]
+ifeval::["{Build}" == "satellite"]
+  * Synchronized content repositories for Red{nbsp}Hat Enterprise Linux. For more information, see link:/html/content_management_guide/importing_red_hat_content#Importing_Red_Hat_Content-Synchronizing_Red_Hat_Repositories[Synchronizing Red{nbsp}Hat Repositories] in the _Content Management Guide_.
+endif::[]
   * A {SmartProxyServer} managing the network for bare metal hosts. For unattended provisioning and discovery-based provisioning, {ProjectServer} requires PXE server settings. For more information, see xref:Configuring_Networking[].
+ifeval::["{Build}" == "satellite"]
   * An activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
-  * A blank bare metal host.
+endif::[]
+ifeval::["{Build}" == "foreman"]
+  * If the Katello plugin is installed, an activation key for host registration. For more information, see link:/html/content_management_guide/managing_activation_keys#Managing_Activation_Keys-Creating_an_Activation_Key[Creating An Activation Key] in the _Content Management_ guide.
+endif::[]
+  * A bare metal host or a blank VM.
+
+NOTE: PXE workflows can be used to provision virtual machines which are not under {Project} control as well.
```

Comment:
"Blank bare metal host" is misleading. You can provision hosts which are not blank (like empty). Also VMs can be provisioned as well.